packages/shared/lib/services/sync/job.service.ts

```diff
@@ -193,3 +193,15 @@ export async function hardDeleteJobs({ syncId, limit }: { syncId: string; limit:
             sub.select('id').from('_nango_sync_jobs').where({ sync_id: syncId }).limit(limit);
         });
 }
+
+export async function deleteJobsByDate({ expiresAfterDays, limit }: { expiresAfterDays: number; limit: number }): Promise<number> {
```

Comment:
`deleteJobsOlderThan` is a better name I think, conveying how the date is being used